.gitignore

```diff
@@ -0,0 +1,5 @@
+# .gitignore
+node_modules/
+
+# files
+test.md
```

Comment:
Great job adding the .gitignore file, it's working as requested 👍 . But you forgot to use a README template as requested for this project. You can find the template on this link and customize it to fit this project: https://github.com/microverseinc/readme-template
Make sure to add it to your root folder.

This will help your project look more professional and it might catch the eye of people that take a look at your project, like recruiters.